content/blog/2023-04-04-the-pci-device-that-wasnt.md

```diff
@@ -84,9 +84,9 @@ So `pci_notify()` gets called with our VIO device (somehow), and we're convertin
 Now we know why and how we're blowing up, but we still don't understand how we got here, so let's back up further.
 
 
-# Notifiers
+## Notifiers
 
-The kernel's device subsystem allows consumers to register callbacks to be notified on given events.  I'm not going to go into a ton of detail on how they work, because I don't fully understand myself, and there's a lot of internals of the device subsystem involved.
+The kernel's device subystem allows consumes to register callbacks so that they can be notified of a given event.  I'm not going to go into a ton of detail on how they work, because I don't fully understand myself, and there's a lot of internals of the device subsystem involved.
```

Comment:
regressions: `subsystem` -> `subystem`, `consumers` -> `consumes`

Reply:
https://github.com/sthbrx/sthbrx-blog/pull/22#discussion_r1156709473 copy pasted what rohan wrote with the assumption it was an improvement